domain/consensus/processes/reachabilitymanager/reindex_stretch_test.go

```diff
@@ -0,0 +1,129 @@
+package reachabilitymanager_test
+
+import (
+	"fmt"
+	"github.com/kaspanet/kaspad/domain/consensus"
+	"github.com/kaspanet/kaspad/domain/consensus/model/externalapi"
+	"github.com/kaspanet/kaspad/domain/dagconfig"
+	"github.com/kaspanet/kaspad/infrastructure/logger"
+	"math"
+	"math/rand"
+	"os"
+	"testing"
+)
+
+// Test configuration
+const (
+	numBlocksExponent = 12
+	logLevel          = "warn"
+	validateMining    = false
+)
+
+func LoadJsonDAG(t *testing.T, fileName, testName string, addArbitraryBlocks, useSmallReindexSlack bool) {
+	t.Parallel()
+
+	logger.SetLogLevels(logLevel)
+	params := dagconfig.SimnetParams
+	params.SkipProofOfWork = true
+	tc, teardown, err := consensus.NewFactory().NewTestConsensus(&params, false, testName)
+	if err != nil {
+		t.Fatalf("Error setting up consensus: %+v", err)
+	}
+	defer teardown(false)
+
+	if useSmallReindexSlack {
+		tc.ReachabilityManager().SetReachabilityReindexSlack(10)
+	}
+
+	f, err := os.Open(fileName)
```

Comment:
Please gzip the json files. (Just gzip, without tar)
Then you can utilize [gzip.Reader](https://golang.org/pkg/compress/gzip/#Reader) to read the file seamlessly.

Talk to me if you are not sure what I want from you.

Reply:
Done. Please make sure is was done correctly.